src/resolvers/resolver.ts

```diff
@@ -12,20 +15,36 @@ const resolvers = {
       const profiles = await Profile.find({});
       return profiles;
     },
+
+    async getAllCohorts() {
+      const allCohorts = await Cohort.find({}).populate('members').populate("coordinator");
+      return allCohorts;
+    },
+
+    async getCohort(_:any, { cohortName }: any) {
+      const getCohort = await Cohort.findOne({ name:cohortName }).populate('members').populate("coordinator");
```

Comment:
Please change the `variable name`, you are referencing a function or a `getter` as an object.

You can make it 
``` 
const cohort = await ....
// then check
if (!cohort)
// instead of 
if (!getCohort)
```

Reply:
> Please change the `variable name`, you are referencing a function or a `getter` as an object.
> 
> You can make it
> 
> ```
> const cohort = await ....
> // then check
> if (!cohort)
> // instead of 
> if (!getCohort)
> ```

these are just dummy cohorts and will be removed once the task of programs and cohorts is merged